### PR TITLE
Update study.txt

### PR DESCRIPTION
**Pull request by dominikl**
The study.txt was basically empty. Added some information from the respective published article.

### CODE REVIEW

File: idr0053-study.txt [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+Study Person Address				
+Study Person ORCID				
+Study Person Roles				
+				
+# Study License and Data DOI				
+Study License	CC-BY 4.0			
+Study License URL	https://creativecommons.org/licenses/by/4.0/			
+Study Copyright	Faas et al.			
+Study Data Publisher	University of Dundee			
+Study Data DOI
```
Comment (sbesson) [inline idr0053-study.txt]: A DOI had minted for this dataset by JCB (10.1083/jcb.201201140.dv). We agreed with the submitter the DOI should be redirected to the new IDR URL.

File: idr0053-study.txt [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+# Study Contacts				
+Study Person Last Name	Faas			
+Study Person First Name	Frank			
+Study Person Email				
+Study Person Address				
+Study Person ORCID				
+Study Person Roles				
+				
+# Study License and Data DOI				
+Study License	CC-BY 4.0
```
Comment (sbesson) [inline idr0053-study.txt]: Previous IDR datasets already coming from JCB (idr0004) were published with `CC-BY-NC-SA`. Might be something we want to clarify

File: idr0053-study.txt [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+Study Description	A key obstacle in uncovering the orchestration between molecular and cellular events is the vastly different length scales on which they occur. We describe here a methodology for ultrastructurally mapping regions of cells and tissue as large as 1 mm(2) at nanometer resolution. Our approach employs standard transmission electron microscopy, rapid automated data collection, and stitching to create large virtual slides. It greatly facilitates correlative light-electron microscopy studies to relate structure and function and provides a genuine representation of ultrastructural events. The method is scalable as illustrated by slides up to 281 gigapixels in size. Here, we applied virtual nanoscopy in a correlative light-electron microscopy study to address the role of the endothelial glycocalyx in protein leakage over the glomerular filtration barrier, in an immunogold labeling study of internalization of oncolytic reovirus in human dendritic cells, in a cryo-electron microscopy study of intact vitrified mouse embryonic cells, and in an ultrastructural mapping of a complete zebrafish embryo slice.		 	
+Study Organism	Danio rerio, Mus musculus, Homo sapiens
```
Comment (sbesson) [inline idr0053-study.txt]: This is the first time we have a multiple organism study right? Should these (and the term source REF/Accession lines below) be separate by tabs?
In terms of project annotation, I assume we will have 3 K/V pairs with the same key

**Review by sbesson — COMMENTED** [review]
Thanks for starting to fill that in a. A few immediate questions.

File: idr0053-study.txt (hunk 3 above)
Comment (dominikl) [reply idr0053-study.txt]: I think so, couldn't find any previous study which had multiple organisms. You're right, should be separated by tabs in that case. Hope duplicate keys won't be a problem for the project level map annotations.

Comment (sbesson) [reply idr0053-study.txt]: Should not be. We already have duplicate keys e.g. in the case of multiple publications associated with a study for instance.

**Comment by dominikl** [comment]
- Changed license, added doi and separated organisms by tabs.
- Added @francesw 's annotation.csv.

**Comment by sbesson** [comment]
thanks. Merging as @francesw has follow-up changes on the study and annotation files.

**Pull request merged**